picard/ui/infodialog.py

```diff
@@ -253,7 +253,7 @@ def format_file_info(file_):
         info.append((_('Format:'), file_.orig_metadata['~format']))
     try:
         size = os.path.getsize(encode_filename(file_.filename))
-        sizestr = "%s (%s)" % (bytes1human.decimal(size), bytes2human.binary(size))
```

Comment:
Damn, no idea how it happened, it was just moving existing code around (in https://github.com/metabrainz/picard/commit/096e3ef2a05b0ddea4eb179b398b762061ec36fc#diff-bac36197117fdc03a66aae48f5f9e570)

Good catch !